src/System.Private.CoreLib/shared/System/IO/Stream.cs

```diff
@@ -189,6 +189,65 @@ private int GetCopyBufferSize()
             return bufferSize;
         }
 
+        public virtual void CopyTo(ReadOnlySpanAction<byte, object> callback, object state, int bufferSize) =>
+            CopyTo(new WriteCallbackStream(callback, state), bufferSize);
+
+        public virtual Task CopyToAsync(Func<object, ReadOnlyMemory<byte>, CancellationToken, Task> callback, object state, int bufferSize, CancellationToken cancellationToken) =>
```

Comment:
Swap the first two parameters in the `Func<...>`. See the comment https://github.com/dotnet/corefx/issues/21472#issuecomment-332283266 for the approved API.

Reply:
I agree. Fixed.